glibc.yaml

```diff
@@ -1,7 +1,7 @@
 package:
   name: glibc
-  version: 2.40
-  epoch: 40
+  version: "2.41"
+  epoch: 1
```

Comment:
epoch: 0 ?

Reply:
we had; and withdrew -r0 build before, didn't we in #42251? we shouldn't reuse that epoch.

Reply:
It's 1 on purpose! ;) I could have explained it in the description, but we already had `2.41-r0` in the repository and withdrew it due to issues with gcc. So it has to be 1.